docs/source/tutorials/Visualisation_assembly_reads_pileup_IGV.md

```diff
@@ -0,0 +1,96 @@
+# Visualizing assembly and associated reads pileup with IGV
+
+ The [IGV: Integrative Genomics Viewer](http://software.broadinstitute.org/software/igv/home) software allows to visulally explore and scroll through sequences, that may or not be splitted in contigs, and may or not be associated with reads pileup. It can eg. allow you to detect specific positions were variants are errorneous (eg. at the end of contigs where reads are poorly mapped) and visualize eg. locations where coverage is not homogeneous (ie. repeated regions that are not proprely assembled).
+
+ NB: Convention: means `<change_me>`
+
+## 1. Getting IGV
+
+- you can install IGV on your pc do so.
+- you can use SAGA with a graphical user interface
+
+### 1.1. Installing IGV on your PC
+If want to do that at your PC you can install IGV as such
+
+<div style="background-color:silver">
+
+_**PRACTICAL EXERCISE:**_
+
+```bash
+conda create -n <envname> -c bioconda igv
+conda activate <IGV:envname>
+```
+</div>
+
+### 1.2. Connect to SAGA with a graphical interface
+
+Please see the techinical: [Using a graphical window in SAGA](../technical/Saga_graphical_interface.md)
+
+## 2. Example: Using IGV with reads mapped to its own assembly
+
+We have assembled reads and annotated using Bifrost pipeline. The data for today's practical are found in  `/cluster/projects/nn9305k/tutorial/20210412_mapping_visualization/visualisation_files`
```

Comment:
remove today

Reply:
Done - but hanging on data is/are ... RRR
so "read" (on the fly) https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular
Ok to you are ?